### ISSUE TITLE
QnA Maker throws an exception when calling GetAnswersAsync

### ISSUE THREAD

**Issue by rob-derosa** [microsoft/botbuilder-dotnet#2038]
## Version
4.4.4

## Describe the bug
The QnAMaker SDK errs out when asking a specific question.

```
System.ArgumentException: An item with the same key has already been added. Key: question
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
   at Microsoft.Bot.Builder.AI.QnA.QnAMaker.FillQnAEventAsync(QueryResult[] queryResults, ITurnContext turnContext, Dictionary`2 telemetryProperties, Dictionary`2 telemetryMetrics, CancellationToken cancellationToken) in d:\a\1\s\libraries\Microsoft.Bot.Builder.AI.QnA\QnAMaker.cs:line 256
   at Microsoft.Bot.Builder.AI.QnA.QnAMaker.OnQnaResultsAsync(QueryResult[] queryResults, ITurnContext turnContext, Dictionary`2 telemetryProperties, Dictionary`2 telemetryMetrics, CancellationToken cancellationToken) in d:\a\1\s\libraries\Microsoft.Bot.Builder.AI.QnA\QnAMaker.cs:line 202
   at Microsoft.Bot.Builder.AI.QnA.QnAMaker.GetAnswersAsync(ITurnContext turnContext, QnAMakerOptions options, Dictionary`2 telemetryProperties, Dictionary`2 telemetryMetrics) in d:\a\1\s\libraries\Microsoft.Bot.Builder.AI.QnA\QnAMaker.cs:line 192
   at Botflix.Bot.Dialogs.MainDialog.OnQuestionAskedAsync(DialogContext dc, Intent intent, CancellationToken cancellationToken) in /Volumes/Data/Projects/Botflix/src/Botflix.Bot/Dialogs/Main/MainDialog.cs:line 107
   at Botflix.Bot.Dialogs.MainDialog.RouteAsync(DialogContext dc, CancellationToken cancellationToken) in /Volumes/Data/Projects/Botflix/src/Botflix.Bot/Dialogs/Main/MainDialog.cs:line 74
```

## To Reproduce
1. Make a call to `GetAnswersAsync` using KB: 4aac4281-9f05-4122-97a8-34aa51bbfb1e with the question "get billing history:
2. Error specified above will be thrown.

## Expected behavior
No error

[bug]

**Comment by garypretty** [comment]
This is a duplicate of #1903 I believe.  This has been addressed and merged a few days ago in PR #1899

**Comment by gabog** [comment]
Closing as per Gary's comment.
